### DISCUSSION TITLE
make table responsive like the screenshot

### DISCUSSION THREAD

**Discussion by SamerNajadaUI** [tailwindlabs/tailwindcss#12859] [Help]
Hi all , i want to make hovering table responsive web and mobile like this can anyone help me with this
![image](https://github.com/tailwindlabs/tailwindcss/assets/136931204/e853e6fb-00b3-41cb-96b5-326420d3d7c6)
![image](https://github.com/tailwindlabs/tailwindcss/assets/136931204/048ee472-06ff-4045-9c04-decacbe2a51e)
this is mobile responsive will be
![image](https://github.com/tailwindlabs/tailwindcss/assets/136931204/78be6831-13cb-4186-bb14-0446fbf0dc2e)
i tried to make it but it's not work well with me : https://play.tailwindcss.com/OXNEPvMYN9

**Comment by SamerNajadaUI** [comment]
up

**Comment by SamerNajadaUI** [comment]
1

**Comment by giacomoferretti** [comment]
I think this is not the proper place to ask this question.
